docs/design/python/examples/example_client.py

```diff
@@ -0,0 +1,109 @@
+""" The example_client shows the canonical shape of a simple non-hierarchical sync client
+
+This particular example is intended to run on both Python 2.7 and 3.5+.
+"""
+
+import azure.core as core
+import azure.core.transport as core.transport
```

Comment:
I have honestly never seen anyone use `as` for a dotted name before, but based on how the bytecode works I can see how it wouldn't fail. Typically, though, people would use an underscore, e.g.
```suggestion
import azure.core.transport as core_transport
```

Reply:
Sorry, the example_client.py should not be included in the PR. It was not ready. So, yeah, not the best example... at all...